### PR TITLE
Display PDU Session Container on any GTP-U message.

### PR DESCRIPTION
**Pull request by TetsuyaMurakami**
This is just a checklist to guide you. You can remove it safely.

**Checklist:**

-   [O] If you are new to Scapy: I have checked <https://github.com/secdev/scapy/blob/master/CONTRIBUTING.md> (esp. section submitting-pull-requests)
-   [O] I squashed commits belonging together
-   [O] I added unit tests or explained why they are not relevant
-   [O] I executed the regression tests for Python2 and Python3 (using `tox` or, `cd test && ./run_tests_py2, cd test && ./run_tests_py3`)

> brief description what this PR will do, e.g. fixes broken dissection of XXX
For 5G service, PDU session container can be added on not only regular User Data but also End Marker message. PDU Session Container can be displayed on only User Data packet. This PR can allow to display PDU Session Container on any GTP-U message.

> if required - short explanation why you fixed something in a way that may look more complicated as it actually is

> if required - outline impacts on other parts of the library
None

fixes #xxx (add issue number here if appropriate, else remove this line)

### CODE REVIEW

File: scapy/contrib/gtp.py [hunk 1]
```diff
@@ -258,19 +258,21 @@ class GTP_U_Header(GTPHeader):
     def guess_payload_class(self, payload):
         # Snooped from Wireshark
         # https://github.com/boundary/wireshark/blob/07eade8124fd1d5386161591b52e177ee6ea849f/epan/dissectors/packet-gtp.c#L8195  # noqa: E501
+        if self.E == 1:
+            if self.next_ex == 0x85:
+                return GTPPDUSessionContainer
+            else
```
Comment (guedou) [inline scapy/contrib/gtp.py]: This triggers PEP08 errors. See https://travis-ci.com/secdev/scapy/jobs/269661800#L655

**Comment by guedou** [comment]
Thanks for this PR. Could you provide an example of the problem that it is solving?

File: scapy/contrib/gtp.py (hunk 1 above)
Comment (TetsuyaMurakami) [reply scapy/contrib/gtp.py]: I fixed the style.

**Comment by TetsuyaMurakami** [comment]
> Thanks for this PR. Could you provide an example of the problem that it is solving?

Originally PDU Session Container can be displayed only if GTP type is 255 (User Data). However, according to 5G definition, PDU Session Container can be used in not only User Data (GTP type is 255) but also End Marker message (GTP type is 254). Hence, this PR can allow to display PDU Session Container regardless of GTP type if PDU Session Container is present on the received packets.

**Pull request merged**